application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -1,5 +1,6 @@
 package org.togetherjava.tjbot.features.reminder;
 
+import com.sun.source.tree.OpensTree;
```

Comment:
whats that import? looks odd

Reply:
It wasn't supposed to be there, since the imports were minimized it slipped through. Fixed it after installing sonarLint plugin.